src/test/java/com/iota/iri/network/spam/DropFixedPercentageTest.java

```diff
@@ -0,0 +1,128 @@
+package com.iota.iri.network.spam;
+
+import com.iota.iri.network.Neighbor;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class DropFixedPercentageTest {
+
+    @Test
+    public void givenNewInstanceWhenCheckBroadcastThenTrue() {
+        SpamPreventionStrategy strategy = new DropFixedPercentage(33, new ArrayList<>());
+        assertTrue("No neighbor is spamming yet.", strategy.broadcastTransactionFrom(mock(Neighbor.class)));
+    }
+
+    @Test
+    public void givenNoHistoryWhenCheckBroadcastThenTrue() {
+        Neighbor a = neighbor(0, 42);
+        SpamPreventionStrategy strategy = new DropFixedPercentage(100, Collections.singletonList(a));
+        assertTrue("not spamming", strategy.broadcastTransactionFrom(a));
```

Comment:
ditto (also look for your other tests)

Reply:
Yes, same as above but with one neighbor. Probably we can delete the test givenNewInstanceWhenCheckBroadcastThenTrue as it doesn't add value.